tests/ops/op_math/test_sum.py

```diff
@@ -1062,7 +1062,7 @@ def circuit(weights):
         assert qnp.allclose(grad, true_grad)
 
     def test_non_hermitian_op_in_measurement_process(self):
-        """Test that non-hermitian ops in a measurement process will raise a warning."""
+        """Test that non-hermitian ops in a measurement process will fail."""
```

Comment:
If we specify `interface=None` on the qnode, then it will execute.  The only reason it was failing was because it was using the autograd device and assuming we needed to differentiate it.

Reply:
nice one. updated